src/parser.cpp

```diff
@@ -521,21 +521,15 @@ options_metadata parser::load_settings()
 
 config::checkpoint::list parser::default_checkpoints() {
     //case config::settings::mainnet:
-
-// #ifdef BITPRIM_LITECOIN
-//     auto const testnet = (configured.network.identifier == 4056470269u); //Litecoin
-// #else
-//     auto const testnet = (configured.network.identifier == 118034699u);  //Bitcoin
-// #endif //BITPRIM_LITECOIN
-    // bool const testnet = is_testnet(configured.network.identifier, configured.network.bitcoin_cash);
-
-    
+   
 
     config::checkpoint::list checkpoints;
 
+//TODO(fernando): Set Litecoin checkpoints
 #if defined(BITPRIM_CURRENCY_BCH)
-    if (get_network() == config::settings::testnet) {
-        checkpoints.reserve(15);
+    if (get_network(configured.network.identifier) == config::settings::testnet) {
```

Comment:
I would separate each of the forks in its own inline function for better readability